apps/src/aichat/views/modelCustomization/ModelDescriptionPanel.tsx

```diff
@@ -55,18 +32,19 @@ const ModelDescriptionPanel: React.FunctionComponent<{
   const shouldScroll = userWantsScroll && contentNeedsScroll;
 
   const selectedModel =
```

Comment:
Could we just pass in the whole `model` object so we don't need this lookup/fallback logic here?

Reply:
+1, was thinking the same thing :)

Reply:
Good idea!

Reply:
So, the problem with that is the id is a piece of state that we're calling onChange on in the SimpleDropdown and it needs to be a string. I can't think of a good way to make it work without just moving that lookup somewhere else. But I'm also feeling kinda brain mush at the moment so I might be missing something obvious.

Reply:
Nice! Would we still need `availableModels` passed down as a prop?
And when a prop is needed at multiple levels of components, just curious when a`context` would be appropriate to use.

Reply:
Yeah, that's true. Removing the constraint of having the left and right selected states in `CompareModelsDialog` does make it simpler. Thanks for mapping out the impl! I'll do that refactor. 

Re: Alice
I think we'll still need `availableModels` passed in to populate the dropdown, since we need both id and name for that.
From what I just read about `context` (thanks for mentioning that so I can learn about it :)) Seems like it would be way over-kill for this situation & the use-cases described [here](https://react.dev/learn/passing-data-deeply-with-context#before-you-use-context) are quite broad. But I'm not sure where the line is. I see a bunch of uses of `useContext` in our code base in more specific cases. 🤷🏻